packages/vx-event/test/localPoint.test.ts

```diff
@@ -1,7 +1,49 @@
 import { localPoint } from '../src';
+import localPointGeneric from '../src/localPointGeneric';
 
 describe('localPoint', () => {
   test('it should be defined', () => {
     expect(localPoint).toBeDefined();
   });
+
+  test('it should return null if called with no arguments', () => {
+    // @ts-ignore
+    expect(localPoint()).toBeNull();
+    // @ts-ignore
+    expect(localPointGeneric(document.createElement('div'))).toBeNull();
+  });
+
+  test('it should handle localPoint(event) and get node from event.target', () => {
+    const e = new MouseEvent('test', {
+      clientX: 0,
```

Comment:
only thought on these is if we should update the values to be non-zero since `0` is the default behavior of `localPoint`